qemu/target/ppc/excp_helper.c

```diff
@@ -1061,7 +1061,11 @@ void helper_store_msr(CPUPPCState *env, target_ulong val)
 }
 
 #if defined(TARGET_PPC64)
+#if defined(_MSC_VER) && defined(__clang__)
+void helper_pminsn(CPUPPCState *env, uint32_t insn)
```

Comment:
This looks super weird... `powerpc_pm_insn_t` is just an `int` and I don't think providing a different function signature is good. Could you post the error message related given by `clang-cl`?